.travis.yml

```diff
@@ -7,11 +7,12 @@ julia:
   - 1.0.4
   - 1.1
   - 1.2
+  - 1.3
   - nightly
 
 matrix:
   allow_failures:
-    - julia: nightly
+    # - julia: nightly
```

Comment:
I'd rather keep this as failure can happen later and it's annoying to have to change it back.